### PR TITLE
add page for capturing cancel recommendation rational

### PR DESCRIPTION
**Pull request by xebu**

### CODE REVIEW

File: server/controllers/recommendation/spoCancelRecommendationController.test.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        },
+        token: 'token1',
+      },
+    })
+    const next = mockNext()
+    await spoCancelRecommendationController.get(mockReq(), res, next)
+    expect(res.locals.page).toEqual({ id: 'cancelDntrRationale', bodyText: 'cancel this decision not to recall' })
+    expect(res.render).toHaveBeenCalledWith('pages/recommendations/spoCancelRecommendationRationale')
+  })
+  it('present without data when in NO_RECALL_DECIDED state', async () => {
```
Comment (DarrenW-HMPPS) [inline server/controllers/recommendation/spoCancelRecommendationController.test.ts:42]: Is this test name right? It's the same as the previous one, should it be 
"present without data when in PO_START_RECALL state"?

File: server/middleware/featureFlags.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const featureFlagsDefaults: Record<string, FeatureFlagDefault> = {
     default: false,
   },
   flagCancelRecommendationProcess: {
-    label: 'Allow process workflow for delete recommendation',
+    label: 'Allow process workflow for cancel recommendation',
     description: 'Shows banner for deleting a recommedation for SPO/ACO users',
```
Comment (DarrenW-HMPPS) [inline server/middleware/featureFlags.ts:31]: Should "deleting" be updated to "cancelling" to match the change in label?

File: server/textStrings/en.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export const strings: Record<string, Record<string, string>> = {
   errors: {
     noSpoRecallTypeSelected: 'Select whether you have decided to recall or made a decision not to recall',
+    missingSpoCancelRecommentationRationale: 'You must explain your decision',
```
Comment (DarrenW-HMPPS) [inline server/textStrings/en.ts:4]: Typo in the word "Recommentation"

File: server/views/pages/recommendations/spoCancelRecommendationRationale.njk [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                }) }}
+            {% endset %}
+            <form novalidate method="post">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+                <input type="hidden" name="crn" value="{{ recommendation.crn }}"/>
+
+                {{ cancelRationaleHtml | safe }}
+                <p class='govuk-button-group'>
+
+                    {# {{ govukButton({
```
Comment (DarrenW-HMPPS) [inline server/views/pages/recommendations/spoCancelRecommendationRationale.njk:37]: Commented out code should probably be removed

**Review by DarrenW-HMPPS — CHANGES_REQUESTED** [review]

File: server/views/pages/recommendations/spoCancelRecommendationRationale.njk (hunk 4 above)
Comment (xebu) [reply server/views/pages/recommendations/spoCancelRecommendationRationale.njk]: The one that got away. Removed. It's for the next ticket.

**Review by DarrenW-HMPPS — APPROVED** [review]

**Pull request merged**